src/tests/nativeaot/SmokeTests/ComWrappers/ComWrappersNative.cpp

```diff
@@ -12,7 +12,7 @@
 #define DLL_EXPORT extern "C" __attribute((visibility("default")))
 #endif
 
-#if !defined(__stdcall)
+#ifndef TARGET_WINDOWS
```

Comment:
I swear that I copy that from some place. I not that creative in C++ to think about such condition. Maybe this should be changed in other places as well.

Reply:
https://github.com/dotnet/runtime/pull/62763/files#diff-853460ee46510792e4ae93dd6307874a18b0265119a3a78d044d60fd74e13626R18 in PInvokeNative.cpp same construct

Reply:
Yeah, the one in PInvokeNative.cpp looks also wrong but it's not a build break on x86 (the one in this file was a build break because calling convention of the override doesn't match the base method since the macro turns `__stdcall` into nothing).